CONTRIBUTING.md

```diff
@@ -1,8 +1,8 @@
 # Contributing guide
 
 
-Development environment
-------------
+##Development environment
```

Comment:
This formatting does not seem to be working, space after `##` is missing:
<img width="978" alt="screen shot 2018-06-05 at 09 55 17" src="https://user-images.githubusercontent.com/1409983/40959863-9d491ad8-68a6-11e8-83e9-769638d4e568.png">